src/App.tsx

```diff
@@ -2,24 +1,57 @@
+import React, { useState } from 'react';
+
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const total = 42;
+const startingPage = 5;
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(startingPage);
```

Comment:
The initial state for 'currentPage' is set to 1, which is correct. However, the initial state for 'perPage' is using 'startingPage' which is a constant set to 5. This could be confusing as 'perPage' suggests the number of items per page, not the starting page number. Consider renaming 'startingPage' to something like 'initialItemsPerPage' for better clarity.

Reply:
Yeah, my bad, I admit